core/server/src/main/java/alluxio/worker/netty/DataServerReadHandler.java

```diff
@@ -265,6 +278,7 @@ private void reset() {
     } finally {
       mLock.unlock();
     }
+    mEOFSent.set(false);
```

Comment:
Is it guaranteed that all `replySuccess` calls will be handled before this is set?